templates/taxi/driver_list.html

```diff
@@ -6,8 +6,14 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+          {% if driver.id == user.id %}
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+            (Me)
+          {% else %}
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
           ({{ driver.first_name }} {{ driver.last_name }})
+          {% endif %}
+
         </li>
```

Comment:
Almost the same code diff is only (Me)

Reply:
Solved this using if else, is it correct?